game-headed-javafx/src/main/java/org/triplea/game/client/ui/javafx/TripleA.java

```diff
@@ -40,10 +37,6 @@ public void start(final Stage stage) {
     navigationPane.switchScreen(FxmlManager.MAIN_MENU_PANE);
 
     setupStage(stage, scene, loadedNode.getController());
-    // Don't invoke Swing if headless (for example in tests)
-    if (!GraphicsEnvironment.isHeadless()) {
-      SwingUtilities.invokeLater(GameRunner::newMainFrame);
-    }
```

Comment:
@RoiEXLab this was removed as it does not seem to belong in the JavaFx componentry. This invocation seemed like an intermediary step to get java fx working. I'm hoping we can circle back and get this fixed correctly and not have swing dependencies in JavaFx, all in part of fixing up the game startup models.

I tried to test if this had an impact, and was getting an error launching this class:

> Error: JavaFX runtime components are missing, and are required to run this application

Reply:
Did you launch this class via the new RunConfiguration?
There seems to be an issue with JavaFX where it can't find the components when the main class extends application and is not a module

Reply:
Ah, thanks for pointing out the run configuration. :+1: 
The JavaFx variant launches both before and after this update. Admittedly that was not a very thorough regression check.

Reply:
Try launching a game once.
The reason I introduced those 3 lines of code is because originally some swing code I invoke at some point actually expects this mainframe to exist, otherwise an NPE will be thrown.
The reason for the headless check was because we initially had some javafx integration tests that basically clicked through the screens, but this framework is only able to create dummy javafx components, so swing just crashed the test.